pkg/odo/genericclioptions/clientset/clientset.go

```diff
@@ -125,7 +134,12 @@ func Fetch(command *cobra.Command) (*Clientset, error) {
 	if isDefined(command, PROJECT) {
 		dep.ProjectClient = project.NewClient(dep.KubernetesClient)
 	}
-
+	if isDefined(command, WATCH) {
+		dep.WatchClient = watch.NewWatchClient()
+	}
```

Comment:
Please move this one to the section **Without sub-dependencies**

Reply:
Fixed.